stdlib/public/Darwin/Accelerate/Quadrature.swift

```diff
@@ -0,0 +1,258 @@
+    /// - Parameter relativeTolerance: Requested relative tolerance on the result.
+    public init(integrator: Integrator,
+                absoluteTolerance: Double = 1.0e-8,
+                relativeTolerance: Double = 1.0e-2){
+        
+        self.integrator = integrator
+        integrateOptions.abs_tolerance = absoluteTolerance
+        integrateOptions.rel_tolerance = relativeTolerance
+    }
+    
+    /// The quadrature instance's integration algorithm and relevant properties.
+    public var integrator: Integrator {
+        set {
+            switch newValue {
+            case .qng:
+                integrateOptions.integrator = QUADRATURE_INTEGRATE_QNG
+            case .qag(let pointsPerInterval, let maxIntervals):
+                integrateOptions.integrator = QUADRATURE_INTEGRATE_QAG
+                integrateOptions.qag_points_per_interval = pointsPerInterval.rawValue
+                integrateOptions.max_intervals = maxIntervals
+            case .qags(let maxIntervals):
+                integrateOptions.integrator = QUADRATURE_INTEGRATE_QAGS
+                integrateOptions.max_intervals = maxIntervals
+            }
+        }
+        get {
+            let integrator: Integrator!
+            
+            switch integrateOptions.integrator {
+            case QUADRATURE_INTEGRATE_QNG:
+                integrator = .qng
+            case QUADRATURE_INTEGRATE_QAG:
+                integrator = .qag(pointsPerInterval: QAGPointsPerInterval(rawValue: integrateOptions.qag_points_per_interval)!,
+                                  maxIntervals: integrateOptions.max_intervals)
+            case QUADRATURE_INTEGRATE_QAGS:
+                integrator = .qags(maxIntervals: integrateOptions.max_intervals)
+            default:
+                integrator = nil
+            }
+            
+            return integrator
+        }
+    }
+    
+    /// The quadrature instance's requested absolute tolerance on the result.
+    public var absoluteTolerance: Double {
+        set {
+            integrateOptions.abs_tolerance = newValue
+        }
+        get {
+            return integrateOptions.abs_tolerance
+        }
+    }
+    
+    /// The quadrature instance's requested relative tolerance on the result.
+    public var relativeTolerance: Double {
+        set {
+            integrateOptions.rel_tolerance = newValue
+        }
+        get {
+            return integrateOptions.rel_tolerance
+        }
+    }
+    
+    /// The maximum number of subintervals in the subdivision used by QAG and QAGS integrators.
+    public var maxIntervals: Int {
+        set {
+            integrateOptions.max_intervals = newValue
+        }
+        get {
+            return integrateOptions.max_intervals
+        }
+    }
+    
+    /// Number of points per subinterval. Used by the QAG integrator only; other integrators ignore this value.
+    public var qagPointsPerInterval: QAGPointsPerInterval {
```

Comment:
This var is probably overkill, too. I would leave it out unless someone explicitly needs it.

Reply:
Removed (pending push)

Reply:
Fixed & Pushed.